src/iris/api.py

```diff
@@ -700,6 +710,28 @@ def process_resource(self, req, resp, resource, params):  # pragma: no cover
         if resource.allow_read_no_auth and method == 'GET':
             return
 
+        # Ignore HMAC requirements for custom webhooks
+        if len(req.env['PATH_INFO'].split("/")) > 2 and req.env['PATH_INFO'].split("/")[2] == 'webhook':
```

Comment:
Can we do `if req.env['PATH_INFO'].startswith('/v0/webhooks/')` instead? Should be more efficient than splitting twice and it's less code.